packages/fluentui/react-northstar/test/specs/commonTests/extraConformanceTests.tsx

```diff
@@ -1,7 +1,21 @@
 import * as _ from 'lodash';
 import * as path from 'path';
 import { TestObject } from '@fluentui/react-conformance';
-import { parseDocblock } from '@fluentui/scripts-gulp';
+
+import * as doctrine from 'doctrine';
+
+// NOTE: copied code from '@fluentui/scripts-gulp/src/plugins/util/parseDocblock'
+// - all scripts-* packages use esModuleInterop flag which will cause issues when used in northstar source/test files (they don't have that flag enabled)
```

Comment:
if northstar is the only thing that uses `scripts-gulp` why can't that package just disable the esModuleInterop flag?

Reply:
right. because it uses other `scripts-` packages that use it, which causes the same issue. I was thinking about this for some time, but the I dont think the effort of migrating whole /scripts` to stop using `esModuleInterop` is worth the cost ( also there might be other issues from 3rd party typings POV (webpack and friends).